docs/source/gbd2017_models/concept_models/vivarium_gates_bep/concept_model.rst

```diff
@@ -0,0 +1,353 @@
+
+Objective 1: Comparing the cost-effectiveness of BEP, MMN, and IFA
+This objective will compare MMN scenario vs baseline, and IFA vs MMN scenario
+
+Objective 2:  Comparing the cost-effectiveness of universal vs. targeted BEP administration
+This objective will compare scenario with targeted BEP vs scenario with universal BEP
+
+
+.. _3.0:
+
+3.0 Causal framework
+++++++++++++++++++++
+
+.. _3.1:
+
+3.1 Causal diagram
+------------------
+
+  .. image:: causal_dag_bep_scenarios.svg
+
+**Outcome (O)**:
+
+  - (1) birthweight in grams
+  - (2) LAZ score
+  - (3) WLZ score
+
+
+**Effect modifiers**:
+
+  - The effect of BEP on birthweight is differential according to maternal BMI
+
+:underline:`Current evidence scenarios`
+
+• Under the current evidence scenarios, the intervention BEP affects only birthweight. It does so differentially by maternal BMI status (Ota 2015). BMI is an effect modifier on the relationship between BEP and birthweight. 
+
+• BMI should have an effect on LAZ and WLZ scores suggested by the literature (and Gates) but we are not intervening on BMI, and so simply correlatng birthweight and LAZ / WLZ will capture the relationship between BMI and LAZ/WLZ.
+• note x represents a crude effect size because we are not intervening on BMI and we only use this effect in baseline. (ideally crude mean shift birthweight by maternal BMI)  
+• Originally Gates wanted us to model an effect size of RR=2 for BMI on LAZ and WLZ scores, which we did in our februrary model. However, since we are not intervening on BMI, we do not need to model this causal effect. We only want the baseline LAZ and WLZ scores by BMI status and this relationship should be captured by the correlation co-efficient between birthweight and LAZ and WLZ score.
+• BEP only affects pregnancy weight-gain (given during pregnancy at ANC) and not pre-pregnancy BMI, hence no causal arrow from BEP to BMI (pre-pregnancy).
+
+:underline:`Hopes and dreams scenarios`
+
+• Current evidence does not show an effect between BEP and LAZ or WLZ scores (Ota 2015). But there is reason to believe it should affect child growth and hence we are modelling this effect in the hopes and dreams scenario. This is the effect that gates is trying to find in their current trials.
+• Potential reasons why current literature may not capture the effect of BEP on WLZ/LAZ
+  o While the literature has strong evidence there is some causal effect size between birthweight LAZ n WLZ (Harding 2017), the effect from BEP through birthweight might not be big enough to show an effect in LAZ or WLZ through birthweight or the studies might not be powerful enough to detect a difference. 
+  o Following up to child-growth failure outcomes require a long follow-up period, hence this outcome might not be measured accurately in current studies.
```

Comment:
These bullets aren't rendering correctly. I think you need to use "-" only with the appropriate tab indents.